std/container/array.d

```diff
@@ -625,7 +621,13 @@ Complexity: $(BIGOH 1)
         return _data._payload[0];
```

Comment:
It is tested (as for now the unittests are below the class definition, I haven't moved them yet).